judge/migrations/0075_add_user_notes.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2018-10-29 21:00
+from __future__ import unicode_literals
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('judge', '0074_totp'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='profile',
+            name='notes',
+            field=models.TextField(blank=True, help_text='Notes for administrators regarding this user.', null=True, verbose_name='internal notes'),
+        ),
+        migrations.AlterField(
```

Comment:
This shouldn't be in this patch

Reply:
Frankly I attribute this to django generating `AlterField` non-database changes. But in any case, the migration needs to be synced eventually. The convention, however ugly, is to sneak it in with the next actual database change. We should probably work out a better system.